tools/genenotebook/launch_gnb.sh

```diff
@@ -4,22 +4,24 @@ set -e
 
 # Make sure the file always exists even on first grep
 touch mongod.log
+touch mongod_boot.log
 
 echo "Starting mongod, listening on unix socket in $(pwd)"
-mongod --dbpath ./mongo_db/ --unixSocketPrefix "$(pwd)" --bind_ip fake_socket --logpath ./mongod.log --pidfilepath ./mongo.pid &
+mongod --dbpath ./mongo_db/ --unixSocketPrefix "$(pwd)" --bind_ip fake_socket --logpath ./mongod.log --pidfilepath ./mongo.pid > ./mongod_boot.log 2>&1 &
 
 echo "Waiting while mongod starts up"
 
 tries=0
 
-# "Listening on" is for mongodb 5x
+# "Listening on" is for mongodb 100x
 while ! grep -q "Listening on" ./mongod.log; do
 
   tries=$((tries + 1))
 
-  if [ "$tries" -ge 50 ]; then
```

Comment:
I wouldn't change this as it most probably crashes for unknown reason before the loop